AVFoundation/AVCaptureStillImageSettings.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
why do we want to ignore this diagnostic here?

Reply:
This one I will have to check again.